src/hooks/usePatchStatusSelect.ts

```diff
@@ -41,14 +47,16 @@ const reducer = (state: State, action: Action) => {
   }
 };
 
-type FilterSetter = (statuses: string[]) => void;
+export type FilterSetter = (statuses: verionFilters) => void;
 
 type HookResult = [
-  selectedStrings,
-  string[],
-  string[],
+  patchSelectedTasks,
+  verionFilters,
+  verionFilters,
```

Comment:
We are returning a bunch of methods in this hook. It should probably be returning an object instead of returning an array since its easy to accidentally use the wrong index for a field.

Reply:
added to https://jira.mongodb.org/browse/EVG-15414